microsoft-365/security/office-365-security/admin-submission.md

```diff
@@ -12,46 +12,46 @@ search.appverid:
 
 ## How to direct suspicious content to Microsoft for Office 365 scanning
-
 To submit content to Microsoft click the **New submission** button in the top left hand side of the submissions page. A flyout on the right side of the page appears with the option to submit either an email, URL, or file. 
 
 ### Submit a questionable email to Microsoft
 ![Email submission example](../media/submission-flyout-email.PNG)
-1. To submit an email, select **email** and specify the email **network message ID** or upload the email file. 
+1. Admin submissions supports email file submissions with the file types .msg, and .eml formats. It also supports submissions via Network Message ID. To submit an email select **email** and specify the email **network message ID** or upload the email file. In order to save an email as a file [follow these instructions when using Outlook](https://support.office.com/en-us/article/save-a-message-as-a-file-4821bcd4-7687-4d6d-a486-b89a291a56e2). 
 
-2. Specify the recipient(s) that you would like to run the policy check against. The policy check will determine if the email bypassed scanning due to user or tenant level policies. 
+2. Specify the recipient(s) to run the policy check against. The policy check will determine if the email bypassed scanning due to user or tenant level policies. An example would be checking if Safe Senders Policy or an Exchange Transport Rule applied to the specified users. There is also a button to select all recipients to save time.  
 
-3. Specify if the email should have been blocked or not. If the email should have been blocked, specify if it should have been blocked as Spam, Phishing, or Malware. If you are not sure what type it might be, use your best judgement.  
+3. Specify if the email should have been filtered or not. If the email should have been filtered (False Negative) then specify if it should have been filtered as Spam, Phishing, or Malware. If it was blocked and should not have been filtered (False Positive) then select "Should not have been filtered".  
 
-* If the filter was bypassed due to policies upon submission, you'll see information about that policy.
+4. Click the **Submit** button.
 
-* If the filter was not bypassed due to one or more policies, the scan will complete in several minutes. You'll see additional information about the submission by clicking on the status link. This includes the results of the policy check and the rescan verdict. Note this does not run the email through the Office 365 ATP full filtering stack again but runs a partial rescan based on certain attributes of the mail, URL, or file. 
+#### Things to know 
+If the filter was bypassed due to tenant or user settings upon submission information about the policy or setting that allowed or blocked the content will be displayed, but not a rescan result. This is because the original Microsoft verdict was overridden by tenant or user settings. 
```

Comment:
```suggestion
If the filter was bypassed due to tenant or user settings upon submission, information about the policy or setting that allowed or blocked the content will be displayed, but not a rescan result. This is because the original Microsoft verdict was overridden by tenant or user settings. 
```